kex

```diff
@@ -71,14 +71,20 @@ ns_select() {
   fi
 }
 
+po_list_state() {
+  names=($(kubectl $(ns_param) get pods -o=jsonpath='{range .items[*].metadata.name}{@}{"\n"}{end}'))
+  states=($(kubectl $(ns_param) get pods -o=jsonpath='{range .items[*].status.phase}{@}{"\n"}{end}'))
+  for (( i=0; $i < ${#names[@]}; i+=1 )); do echo "${names[i]}: ${states[i]}"; done
+}
+
 po_list() {
   kubectl $(ns_param) \
-    get pods \
-    -o=jsonpath='{range .items[*].metadata.name}{@}{"\n"}{end}'
+     get pods \
```

Comment:
Remove accidental extra single space. There's no official standard, but we're otherwise using 2 spaces for indentation and nesting (not important but anecdotally [google styleguide](https://github.com/google/styleguide/blob/gh-pages/shell.xml#L292) agrees).